packages/core/pegboard-outbound/src/lib.rs

```diff
@@ -0,0 +1,288 @@
+use std::{
+	collections::HashMap,
+	sync::{
+		Arc,
+		atomic::{AtomicBool, Ordering},
+	},
+};
+
+use anyhow::Result;
+use futures_util::{StreamExt, TryStreamExt};
+use gas::prelude::*;
+use namespace::types::RunnerKind;
+use pegboard::keys;
+use reqwest_eventsource as sse;
+use rivet_runner_protocol::protocol;
+use tokio::{sync::oneshot, task::JoinHandle, time::Duration};
+use udb_util::{SNAPSHOT, TxnExt};
+use universaldb::{self as udb, options::StreamingMode};
+
+const OUTBOUND_REQUEST_LIFESPAN: Duration = Duration::from_secs(14 * 60 + 30);
```

Comment:
needs to be configurable on the runner kind. the frontend will have the hardcoded request lifespans to auto-populate for vercel, etc.